web/src/hotkey.js

```diff
@@ -89,6 +89,12 @@ const color_picker_hotkeys = new Set([
     "enter",
 ]);
 
+// A set of characters that, on many keyboard layouts, require modifier keys
+// to type, or where pressing the key with a modifier is expected to produce
+// a different character. We ignore all modifiers when processing these keys,
+// and disallow assigning hotkeys that involve any modifier combined with them.
+const KNOWN_IGNORE_MODIFIERS_KEYS = new Set(["*", "+", "=", "-", "/", ":", "<", ">", "?", "@"]);
```

Comment:
This mechanism is problematic; for example, it causes us to misinterpret `Ctrl`+`+` (browser zoom) as `+` (thumbs up emoji). [Discussion](https://chat.zulip.org/#narrow/channel/9-issues/topic/browser.20zoom.20shortcuts.20not.20working/near/2182431).